dockerfile/cuda11.1.1.dockerfile

```diff
@@ -122,6 +122,18 @@ RUN echo PATH="$PATH" > /etc/environment && \
     echo LD_LIBRARY_PATH="$LD_LIBRARY_PATH" >> /etc/environment && \
     echo SB_MICRO_PATH="$SB_MICRO_PATH" >> /etc/environment
 
+#install STREAM
+RUN cd /tmp && \
+    wget https://download.amd.com/developer/eula/aocc-compiler/aocc-compiler-4.0.0_1_amd64.deb && \
+    apt install -y ./aocc-compiler-4.0.0_1_amd64.deb && \
+    wget https://www.cs.virginia.edu/stream/FTP/Code/stream.c && \
+    /opt/AMD/aocc-compiler-4.0.0/bin/clang -Ofast -mcmodel=large -mavx2 -ffp-contract=fast -lomp -fopenmp -fnt-store=aggressive -DSTREAM_ARRAY_SIZE=400000000 -DNTIMES=10 stream.c -o streamx86.exe && \    
+    /opt/AMD/aocc-compiler-4.0.0/bin/clang -Ofast -mcmodel=large -mavx2 -ffp-contract=fast -march=znver3 -lomp -fopenmp -fnt-store=aggressive -DSTREAM_ARRAY_SIZE=400000000 -DNTIMES=10 stream.c -o streamZen3.exe && \
```

Comment:
can we move the .c code and build logic into third_party/Makefile and keep the aocc-compiler here (maybe move after apt install block)?
we may have a simpler dockerfile for HB series in the future and not need to build stream in cuda/rocm environment

Reply:
Yea let me address.

Reply:
I am still working on addressing this please bare with me

Reply:
I have moved the process to install aocc and build stream to thirdparty directory and it now uses makefile.

Reply:
I will have this addressed by my tomorrow morning. I just need to revert a change I made.

Reply:
addressed